src/tsm/libtsm.h

```diff
@@ -425,6 +451,9 @@ void tsm_vte_set_backspace_sends_delete(struct tsm_vte *vte, bool enable);
 bool tsm_vte_handle_keyboard(struct tsm_vte *vte, uint32_t keysym,
 			     uint32_t ascii, unsigned int mods,
 			     uint32_t unicode);
+bool tsm_vte_handle_mouse(struct tsm_vte *vte, unsigned int cell_x,
```

Comment:
Some documentation here about how this is supposed to be used would be useful. We already don't have API docs, so please at least provide some useful info in the form of comments :)